TourGuide.lua

```diff
@@ -63,8 +63,10 @@ end
 
 function TourGuide:Enable()
 	self.Enable = nil -- Dongle likes to call Enable multiple times if we bug LightHeaded... so we need to nil out to ensure we are only called once
-	-- if TomTom and TomTom.version ~= "SVN" and (tonumber(TomTom.version) or 0) < 120 then self:Print("Your version of TomTom is out of date.  TourGuide waypoints may not work correctly.") end
-
+	if TomTom then 
+		local TomTomVersionNumber = TomTom.version:gsub("v" .. select(4, GetBuildInfo()), ""):gsub("-",""):gsub("%.","")
+		if TomTom.version ~= "SVN" and TomTom.version:find(select(4, GetBuildInfo())) > 0 and (tonumber(TomTomVersionNumber) or 0) < 100 then self:Print("Your version of TomTom is out of date.  TourGuide waypoints may not work correctly.")end
+	end
 	if self.db.char.currentguide == "No Guide" and UnitLevel("player") == 1 and UnitXP("player") == 0 then
```

Comment:
Little bit complicated, but a kinda wanted to have up to date TomTom check that uses the classic version.

Reply:
Current TomTom Classic version number is in from of  v<InterfaceVersion>-<TomTomVersion>
So in short, these changes will check that interface version is the same as current game client and then it will convert TomTomVersion into a number (i.e. current version is 1.0.7 which would convert to 107 which is then compared against number 100, i.e. version 1.0.0)